File: Java_Programming/Day_2/program6.java
// Description : Write a program that prints numbers from 1 to 4 using for loop

class Display
{
    public void display()
    {
        int iCnt = 0;

        for(iCnt = 1 ; iCnt <= 4 ; iCnt++)
        {
            System.out.print(iCnt+"\t");
        }
    }
}

class program6
{
    public static void main(String A[])
    {
        Display dobj = new Display();

        dobj.display();
    }
}

File: CPP_Programming/Day_2/program6.cpp
// Description : Write a program that prints numbers from 1 to 4 using for loop

#include<iostream>
using namespace std;

void Display()
{
    int iCnt = 0;

    for(iCnt = 1 ; iCnt <= 4 ; iCnt++)
    {
        cout<<iCnt<<"\t";
    }

}

int main()
{
    Display();

    return 0;
}